Source/WebCore/rendering/RenderCounter.cpp

```diff
@@ -463,13 +464,14 @@ String RenderCounter::originalText() const
     CheckedPtr child = m_counterNode;
     int value = child->actsAsReset() ? child->value() : child->countInParent();
 
-    String text = listMarkerText(m_counter.listStyle(), value);
+    auto counterStyle = this->counterStyle();
+    String text = listMarkerText(m_counter.listStyleType().type, value, counterStyle.get());
 
     if (!m_counter.separator().isNull()) {
         if (!child->actsAsReset())
             child = child->parent();
         while (CounterNode* parent = child->parent()) {
-            text = listMarkerText(m_counter.listStyle(), child->countInParent())
```

Comment:
small nit that can be addressed later, but I think we should make the third argument of `listMarkerText` non-optional (it can be nullptr if needed though).